examples/permissioned-token/scripts/permissions/kill.js

```diff
@@ -0,0 +1,65 @@
+const { types } = require('@algo-builder/runtime');
+const { issue } = require('../issuance/issue');
+const { executeTransaction, fundAccount } = require('../common/common');
+
+/**
+* NOTE: this function is for demonstration purpose only (if ASA manager is a single account)
```

Comment:
Can you please correct this note everywhere. 

The transaction file doesn't need to be signed and the user can use 2 methods to add his sign:
+ use CLI command `algob sign-multisig`
+ use exported function `signMultiSig`